can-define.js

```diff
@@ -54,7 +54,11 @@ if(process.env.NODE_ENV !== 'production') {
 				value:  "set "+canReflect.getName(obj) + "."+prop
 			});
 		}
-		return Object.defineProperty(obj, prop, definition);
+    var desc = Object.getOwnPropertyDescriptor(obj, prop);
+    if(!desc || desc.writable !== false){
```

Comment:
I think it’d be just a little bit faster if this checked specifically for `undefined` instead of casting `desc`:

```js
if (desc === undefined || desc.writable !== false) {
```

Also, the indentation of these lines should be consistent with the rest of the file.